snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedConceptSearchApiTest.java

```diff
@@ -393,6 +394,18 @@ public void multipleNestedModuleExpand() {
 			.body("items[0].descriptions.items.module.id", allOf(not(emptyIterable()), everyItem(equalTo(Concepts.MODULE_SCT_CORE))))
 			.body("items[0].relationships.items.module.id", allOf(not(emptyIterable()), everyItem(equalTo(Concepts.MODULE_SCT_CORE))));
 	}
+	
+	@Test
+	public void wildcardAcceptLanguageHeader() throws Exception {
```

Comment:
The wildcard header was not added to the test request, it seems 👀

Reply:
Right, fixing.

Reply:
Done